README.md

```diff
@@ -164,6 +174,15 @@ writing defaults...
 $
 ```
 
+## Documentation
+
+Reference documentation of all the options is available [here](https://lnl7.github.io/nix-darwin/manual/index.html#sec-options).--
+This can also be accessed locally using `man 5 configuration.nix`.
+
+`darwin-help` will open a HTML version of the manpage in the default browser.
+
+Furthermore there's `darwin-option` to introspect the settings of a system and its available options.
```

Comment:
Pretty sure that `darwin-option` does not work when using Flakes. I don’t have access to my Mac at the moment, so can’t confirm for sure though. If I’m right, maybe that should be mentioned here.

Reply:
That has also been reported elsewhere. Added a note right after.